projects/httpcomponents-client/FormBodyPartBuilderBuildFuzzer.java

```diff
@@ -0,0 +1,87 @@
+import com.code_intelligence.jazzer.api.FuzzedDataProvider;
+
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.InputStream;
+
+import org.apache.hc.client5.http.entity.mime.ByteArrayBody;
+import org.apache.hc.client5.http.entity.mime.ContentBody;
+import org.apache.hc.client5.http.entity.mime.FileBody;
+import org.apache.hc.client5.http.entity.mime.FormBodyPartBuilder;
+import org.apache.hc.client5.http.entity.mime.InputStreamBody;
+import org.apache.hc.client5.http.entity.mime.StringBody;
+import org.apache.hc.core5.http.ContentType;
+
+public class FormBodyPartBuilderBuildFuzzer {
+    private static final int builderNameLength = 255;
+    private static final int fieldNameLength = 255;
+    private static final int fieldValueLength = 500;
+    private static final int bodyContentSize = 2 * fieldValueLength;
+
+    private enum BodyType { ByteArray, File, InputStream, String }
+
+    public static void fuzzerTestOneInput(FuzzedDataProvider data) {
+        final String builderName = data.consumeString(builderNameLength);
+        final ContentBody contentBody;
+        try {
+            contentBody = generateContentBody(data);
+        } catch (IOException ioe) {
+            // preparations failed ; exit early
```

Comment:
Not a big deal but those comments are generally not needed

Reply:
I personally don't like completely empty catch bodies, but I can change it if you want to.

Reply:
Yes empty catch block are generally a bad practice in projects but are just fine for fuzz target as it is pretty much self-explanatory that we don't want the fuzzer to be triggered on these exceptions. eg: https://github.com/CodeIntelligenceTesting/jazzer/blob/main/examples/src/main/java/com/example/JsonSanitizerValidJsonFuzzer.java

Reply:
So, should I remove the comment or leave it for documentation's sake?